### DISCUSSION TITLE
Debian 12 VM ssh failed

### DISCUSSION THREAD

**Discussion by azzor1337x** [tteck/Proxmox#3019] [Help Needed]
I used a script for Debian 12 VM but the SSH service is "failed". I've tried reinstalling, enabling, editing the config, and several fixes, can you help me? I installed the VM in both Default and other modes, but nothing worked.

**Comment by tteck** [comment]
VM or LXC? I'm guessing that you meant LXC. I just created a Debian 12 LXC using the Advanced settings, set a password, and enabled SSH. I then proceeded to SSH into the newly created LXC without any issues.

**Reply by azzor1337x** [reply]
I'm using the Debian 12 VM script.

**Reply by tteck** [reply]
Did you follow the Post Install instructions?

**Reply by axsrza** [reply]
I'm sorry, but there wasn't.
https://github.com/tteck/Proxmox/discussions/1988
Now it's working perfectly, thanks.
